src/app/components/address-book/address-book.component.html

```diff
@@ -8,7 +8,7 @@
         <mat-card-title>Address Book</mat-card-title>
         <mat-card-content>
             <div
-                *ngIf="!(isMobile$ | async)"
+                *ngIf="!isMobile()"
```

Comment:
Did you also check if this works/changes properly when the screen is resizing?

Reply:
Yes, I did and it works. Change detection also runs on functions

Reply:
I remember having an issue where it would change only in one direction and never change again.